src/meta/src/stream/scheduler.rs

```diff
@@ -156,6 +156,8 @@ where
             return Err(InternalError("fragment has no actor".to_string()).into());
         }
 
+        self.hash_mapping_manager.set_need_consolidation(true);
```

Comment:
Currently we don't have a way to distinguish between fragment creation v.s. fragment reconfiguration in scheduler. Once  we have one, I assume set_need_consolidation should only happen on fragment reconfiguration, right?

Reply:
> Currently we don't have a way to distinguish between fragment creation v.s. fragment reconfiguration in scheduler. Once we have one, I assume set_need_consolidation should only happen on fragment reconfiguration, right?

agree that set_need_consolidation should only happen on fragment reconfiguration